src/currencyservice/server.js

```diff
@@ -169,8 +168,15 @@ function main () {
   const server = new grpc.Server();
   server.addService(shopProto.CurrencyService.service, {getSupportedCurrencies, convert});
   server.addService(healthProto.Health.service, {check});
-  server.bind(`0.0.0.0:${PORT}`, grpc.ServerCredentials.createInsecure());
-  server.start();
+
+  server.bindAsync(
+    `0.0.0.0:${PORT}`,
+    grpc.ServerCredentials.createInsecure(),
+    function() {
+    server.start();
+     logger.info("gRPC server started.")
+    }
```

Comment:
Can we format it to match how we have it in the paymentservice/server.js?
- Update log text to be similar
- Switch log and `server.start()`
- make sure the lines are properly indented
```suggestion
    function() {
        logger.info(`CurrencyService gRPC server started on port ${PORT}`);
        server.start();
    },
```